godotproject/Menus/InGameMenu/InGameMenu.gd

```diff
@@ -91,14 +91,15 @@ func _ready():
 func _process(_delta):
 	var menu_pressed = Input.is_action_just_pressed("menu")
 	# We use whether this node is visible to track whether the menu is open.
-	if menu_pressed && screen_current == null:
+	var inhibit_pause = "inhibit_pause" in get_tree().current_scene && get_tree().current_scene.inhibit_pause
+	if !inhibit_pause && menu_pressed && screen_current == null:
```

Comment:
Since the in-game menu now exists always, we need to check if we block pausing on the current scene, and skip opening the menu in that case.